ecosystem/sep-0006.md

```diff
@@ -614,6 +660,7 @@ The JSON object contains an entry for each Stellar asset that the anchor support
 * `enabled`: `true` if SEP-6 deposit for this asset is supported
 * `authentication_required`: Optional. `true` if client must be [authenticated](#authentication) before accessing the deposit endpoint for this asset. `false` if not specified.
 * `quotes_supported`: Optional. `true` if client can use [SEP-38] quotes to perform a bridge between non-equivalent assets, for instance by depositing fiat BRL in exchange for USDC on the Stellar network. `false` if not specified.
+* `quotes_required`: Optional. `true` if client _must_ use [SEP-38] quotes to perform a bridge between non-equivalent assets, for instance by depositing fiat BRL in exchange for USDC on the Stellar network. `false` if not specified.
```

Comment:
In SEP-31 I was planning on using `quote_required` vs. `quotes_provided` and I'd like to be consistent in SEP-6. If you like `quotes` vs. `quote` I can make sure its the same in SEP-31 too.

Reply:
Since there are two types of quotes, firm and indicative, maybe `quotes_*` would be more accurate? I don't feel strongly towards it though, so we could go with the singular form as well.